ports/quake.2/Quake 2.sh

```diff
@@ -12,16 +13,21 @@ else
   controlfolder="/roms/ports/PortMaster"
 fi
 
+# Source control.txt and tasksetter from PortMaster
+source $controlfolder/device_info.txt
 source $controlfolder/control.txt
 
+
+# Function to get controls from control.txt
 get_controls
 
 GAMEDIR="/$directory/ports/quake2"
 
-if [[ $LOWRES == 'Y' ]]; then
-  swidth="320"
+
+ if [[ $LOWRES == 'Y' && $CFW_NAME != "muOS" ]]; then
```

Comment:
With this you're overriding all 3:2 devices config.
Why hardcode a resolution to a cfw?

Just do `swidth=$DISPLAY_WIDTH` and `sheight=$DISPLAY_HEIGHT` 
to get resolutions from each device.

Reply:
removed

Reply:
removed